data-service/src/main/java/uk/gov/laa/ccms/data/controller/ProceedingController.java

```diff
@@ -30,24 +30,44 @@ public class ProceedingController implements ProceedingsApi {
    * GET Proceedings by category of law, matter type, whether they are enabled,
    * and whether they are amendment-only proceedings.
    *
-   * @param categoryOfLawCode - the category of law code
+   * @param categoryOfLaw - the category of law code
    * @param matterType - the matter type
    * @param enabled - whether the proceeding is enabled
    * @param amendmentOnly - whether the proceeding is amendment-only
    * @param pageable - the pagination settings
    * @return ResponseEntity with the results of the search
    */
   @Override
-  public ResponseEntity<ProceedingDetails> getProceedings(String categoryOfLawCode,
-      String matterType, Boolean amendmentOnly, Boolean enabled, Pageable pageable) {
-    return ResponseEntity.ok(proceedingService.getProceedings(
-            categoryOfLawCode,
-            matterType,
-            enabled,
-            amendmentOnly,
-            pageable));
+  public ResponseEntity<ProceedingDetails> getProceedings(
+      final String categoryOfLaw,
+      final String matterType,
+      final Boolean amendmentOnly,
+      final Boolean enabled,
+      final Boolean lead,
+      final String applicationType,
+      final Boolean larScopeFlag,
+      final Pageable pageable) {
+
+    if (lead != null && lead) {
```

Comment:
Boolean.TRUE.equals(lead), saves checking for null

Reply:
amended